Week-2/poem-webpage.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Poem Webpage</title>
+</head>
+<body>
+    <p>
```

Comment:
Here instead of writing the whole code in single <p> tag, you can create a div and inside this div you can create <P >tags for displaying the data. As p tags are block scope so you no need to add the <br> after every line.

Reply:
I have used div+p tag in the latest commit, please check.